packages/cli-platform-ios/src/commands/runIOS/index.ts

```diff
@@ -48,6 +36,10 @@ function runIOS(_: Array<string>, ctx: Config, args: FlagsT) {
     );
   }
 
+  if (args.configuration) {
+    logger.warn('--configuration has been deprecated. Use --mode instead.');
+  }
```

Comment:
and eventually the same would happen here?

Reply:
Extended warning and assigned config to mode here as well.